src/resources/postcss/views/list/_event.pcss

```diff
@@ -11,6 +11,18 @@
 		@media (--viewport-medium) {
 			margin-bottom: var(--spacer-8);
 			margin-top: var(--spacer-9);
+			margin-left: var(--grid-gutter-small-half-negative);
+			margin-right: var(--grid-gutter-small-half-negative);
+
+			& > .tribe-common-g-col {
+				padding-left: var(--grid-gutter-small-half);
+				padding-right: var(--grid-gutter-small-half);
+
+				@media (--viewport-medium) {
```

Comment:
A few things here:
1. You've got a media query wrapped in a media query here.
2. I don't think we're using padding on mobile right?
3. For desktop, the negative margin should match the padding.

Reply:
1. Nice catch!
2. Yes, no padding on mobile.
3. 👍